rootfs/api/models/app.py

```diff
@@ -171,10 +171,27 @@ def create(self, *args, **kwargs):
             )
 
         # create required minimum resources in k8s for the application
+        namespace = self.id
+        service = self.id
         try:
-            self._scheduler.create(self.id)
+            self.log('creating Namespace {} and services'.format(namespace), level=logging.DEBUG)
+            # Create essential resources
+            try:
+                self._scheduler._get_namespace(namespace)
```

Comment:
Yeah, however we've used scheduling code in the models (like that) for a few months but mostly due to one of my "rewrite the scheduler" PRs ended up in a disaster.

I'll do a diff PR that opens that all up and will expose all `_get`, `_create`, `_update` and `_delete` as I want to have the models interacting with the scheduling client a lot more (like this PR)